pkgs/development/compilers/llvm/7/llvm.nix

```diff
@@ -28,9 +26,6 @@ let
   shortVersion = with stdenv.lib;
     concatStringsSep "." (take 1 (splitString "." release_version));
 
-  inherit
-    (import ../common.nix { inherit (stdenv) lib; })
-    llvmBackendList;
```

Comment:
Do we still need common.nix in that case?

Reply:
No, indeed. Removed.